Python.gitignore

```diff
@@ -101,7 +101,21 @@ ipython_config.py
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
 #poetry.lock
 
-# PEP 582; used by e.g. github.com/David-OConnor/pyflow
+# pdm
+#   Similar to Pipfile.lock, it is generally recommended to include pdm.lock in version control.
+#pdm.lock
+#   pdm stores project-wide configurations in .pdm.toml, but it is recommended to not include it
+#   in version control.
+#   https://pdm.fming.dev/#use-with-ide
+.pdm.toml
+
+# PEP 582; used by e.g. github.com/David-OConnor/pyflow and github.com/pdm-project/pdm
```

Comment:
I think something went wrong. Now we have duplicate lines. Lines 112 to 117 are not needed, you should delete them. Apparently, it's my fault as I added suggestions. I'm sorry.

Reply:
Oops, you're right! I didn't see it before applying, sorry. Now it's fixed